web/src/modules/topic/components/CriteriaTable/CriteriaTable.styles.tsx

```diff
@@ -102,3 +100,10 @@ export const PositionedAddNodeButton = styled(AddNodeButton, options)<AddNodeBut
     }
   }}
 `;
+
+export const StyledTransposeTableButton = styled(Button)`
+  position: absolute;
+  font-size: 0.5em;
```

Comment:
mb for missing this - is this font size actually used for anything? I'm changing it on the button element in the deploy preview and it doesn't seem to affect anything. I would expect this to apply to text on the button, but there isn't any